components/case-study/Conclusion.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="container my-5 py-5">
+    <div class="heading d-flex flex-column">
+      <span class="section__subtext">Conclusion</span>
+      <h2 class="section__heading">
+        Formester - For a Brighter, Efficient and More Effective Future of Lead
+        Generation
+      </h2>
+      <p class="section__description">
+        In conclusion, the adoption of Formester as your lead generation form
+        builder will usher in a new era of lead generation excellence for your
+        company. By recognizing the pivotal role that lead generation forms play
+        in the contemporary digital landscape, your organization will have made
+        a strategic decision by leveraging Formester's powerful capabilities.
+      </p>
+    </div>
+
+    <div class="row gx-5 mt-4">
+      <div v-for="feature in features" :key="feature.title" class="col-md-6">
+        <div class="d-flex flex-column align-items-start mt-4">
+          <nuxt-img
+            :src="`case-study-1/conclusion/${convertToImageName(
```

Comment:
The path is not ideal for putting images. First, it's `hard-coded` so can't be reused. Second (originates from first) tomorrow we have a new case study (if we have more) in that case you will put it in a new folder and then this specific path will not work @dhaladitya108. However something like `case-studies/conclusions` might have worked. But also my suggested name is probably. not perfect

Reply:
removed it